src/vm/page.c

```diff
@@ -245,24 +245,40 @@ void _page_showSizes(void)
 }
 
 
+static unsigned int page_digits(unsigned int n, unsigned int base)
+{
+	return (n < base) ? 1 : 1 + page_digits(n / base, base);
```

Comment:
This algorith can be implemented using iteration. We avoid recursion in the kernel code.

Reply:
Fixed.